app/components/candidate_interface/references_review_component.rb

```diff
@@ -55,7 +55,11 @@ def ignore_editable_for
     end
 
     def too_many_complete_references?
-      references.select(&:feedback_provided?).size > ApplicationForm::MINIMUM_COMPLETE_REFERENCES
+      if FeatureFlag.active?(:reference_selection)
+        false
```

Comment:
I wonder if this should be 

`references.selected.size > ApplicationForm::MINIMUM_COMPLETE_REFERENCES`

This would use this scope 
![image](https://user-images.githubusercontent.com/42515961/119327210-a417fe80-bc7a-11eb-993b-3610a8fe4646.png)

And means we'd still be guarding against having more than 2 in the state that gets sent to the provider.

Wdyt?

Reply:
Yeah interesting one - thinking about this, I'm inclined towards the idea that the system should be enforcing the two selections rule, our existing data should be cleaned up to reflect this, and that we should have an invariant check rather than a user-facing validation.

Maybe it's simpler overall to keep this though, and let them resolve on their own if they somehow get into a state with more than two selected. Will address this as part of application submit!